services/visual_recognition/v3.js

```diff
@@ -139,7 +125,7 @@ module.exports = function (RED) {
         return;
       }
       if (feature !== 'retrieveClassifiersList' && feature !== 'retrieveClassifierDetails' 
-          && feature !== 'deleteClassifier')
+          && feature !== 'deleteClassifier' && feature !== 'deleteAllClassifiers')
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['&&' should be placed at the end of the line.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2712988163/issues/source?bid=3330694&fileBranchId=3344618#l128)